rosidl_generator_c/test/test_interfaces.c

```diff
@@ -527,7 +527,7 @@ int test_unbounded_sequences()
   }
 
   // uint8_values
-  res = rosidl_generator_c__uint8__Sequence__init(&seq->uint8_values, ARR_SIZE);
+  res = rosidl_runtime_c__int8__Sequence__init(&seq->uint8_values, ARR_SIZE);
```

Comment:
No idea how I broke this doing search & replace, but this changes from `uint8` to `int8` results in a compiler warning. See #460.